hbase-protocol-shaded/src/main/protobuf/server/master/Master.proto

```diff
@@ -721,7 +721,8 @@ message ListReplicationSinkServersRequest {
 }
 
 message ListReplicationSinkServersResponse {
-  repeated ServerName server_name = 1;
+  required bool is_replication_server = 1;
```

Comment:
Is this right name for the attribute? The method name for a boolean should be isATTRIBUTE but the ATTRIBUTE itself should not have the 'is' prefix.

Reply:
Oh, you can't change the protobuf index.... it breaks compatibility. Give the replication_server '2' and leave server_name as '1'.

Reply:
This is only for the feature branch. Not merged to any branch now.